packages/extensions/openapi-to-cadl/test/batch/cadl-output/routes.tsp

```diff
@@ -39,19 +39,19 @@ The caller-generated request identity, in the form of a GUID with no decoration
 such as curly braces, e.g. 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0.
 """)
         @header
-        "client-request-id": string;
+        `client-request-id`: string;
```

Comment:
same quotes changes here

Reply:
I noticed some enums have the quotation mark changes as well, not sure if this would be a problem

Reply:
Oh yeah, I didn't notice. I thinks this is the TypeSpec prettier plugin. Good catch let me look deeper into what's going on.

Reply:
The change is because: https://github.com/microsoft/typespec/commit/766cdb728f662c8fdde6417854a301d615ed7cfc#diff-68104c1f8d7943b8b54125fab0d85436006884c0d27ec0cda487d209f9345835